storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -0,0 +1,36 @@
+[id="persistent-storage-csi-azure"]
+= Azure CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk.
+
+:FeatureName: Azure Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
+
+???need info on how to install???To create CSI-provisioned PVs that mount to Azure storage assets, you must manually enable the Azure CSI Driver Operator and the Azure CSI driver for deployment.
+After you enable the deployment, {product-title} installs the operator, driver, and necessary sidecars.
+
+
+* The _Azure CSI Driver Operator_ provides a StorageClass by default that you can use to create PVCs. You also have the option to create the Azure StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-azure.adoc#persistent-storage-using-azure[Persistent storage using Azure].
```

Comment:
We recently updated how to format object references in our guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references

Because of this, you can change the following:
- s/StorageClass by default/storage class by default/
- s/create the Azure StorageClass as described/create the Azure Disk storage class as described

I did notice that this guidance wasn't applied in [AWS EBS](https://docs.openshift.com/container-platform/4.7/storage/container_storage_interface/persistent-storage-csi-ebs.html) docs, so that should probably be changed at some point as well.

Also, since "PVCs" wasn't defined before:

- s/to create PVCs./to create persistent volume claims (PVCs).